x-pack/plugins/enterprise_search/public/applications/app_search/__mocks__/engine_logic.mock.ts

```diff
@@ -8,16 +8,14 @@ import { generatePath } from 'react-router-dom';
 
 export const mockEngineValues = {
   engineName: 'some-engine',
-  // Note: using getters allows us to use `this`, which lets tests
-  // override engineName and still generate correct engine names
```

Comment:
fyi though I plan on adding a `engineBreadcrumbs` selector in a follow up PR sometime next week that will still use a getter. We're passing around engineBreadcrumbs a bunch to child routes and it'd be nice not to have to.

Now I'm debating pulling it out to a utility fn though, it makes importing so nice and flat 🤣